### ISSUE TITLE
Adding more than 4-5 plugins yields error message of "Failed to register plugin client"

### ISSUE THREAD

**Issue by hv10** [falkTX/Carla#1264]
When I run Carla in JACK Multiclient Mode and start to add plugins the error usually shows up when trying to add the fifth plugin (seldomly the sixth).

The error shows up in Carlas internal logs as:
```
Connect: can't connect named futex name = jack_sem.1000_default_Calf Equalizer 5 Band err = Resource temporarily unavailable
Cannot lock down 426 byte memory area (Cannot allocate memory)
Connect: can't connect named futex name = jack_sem.1000_default_Calf Equalizer 5 Band err = Resource temporarily unavailable
Cannot ConnectSemaphore Calf Equalizer 5 Band client
Carla assertion failure: "client != nullptr" in file CarlaEngineJack.cpp, line 1699
```

The JACK Logs suggest that a client was created though:
```
Fri Sep 25 18:48:12 2020: New client 'Calf Envelope Filter' with PID 2623
Fri Sep 25 18:48:13 2020: Client 'Calf Envelope Filter' with PID 2623 is out
```

As my knowledge of the internals of JACK / Carla is very basic I heavily suspect a user / configuration error on my side.
But after reading through multiple issues I could not find someone having a similar problem.

I would be thankful for any help with the problem and please let me know if you require more information.

**Comment by falkTX** [comment]
are you adding the same plugin multiple times, or different ones?

**Comment by hv10** [comment]
I tried both. Loading the same one till I hit the limit and loading different ones.
The issue also happened when I mixed different plugin "manufacturers" (e.g.: CALF & Tracktions Essential Collection)

**Comment by hv10** [comment]
I've checked the specifics again: it happens independent of if I add the same, different (but same maker) or completely independent plugins.

**Comment by hv10** [comment]
Maybe as an additional info: this limitation does not apply to Carlas single client mode.

**Comment by hv10** [comment]
I found the solution to my issue.
As expected it was user / configuration error, not even within Carla but within JACK.

I needed to update the limits of my current user in the `/etc/security/limits.conf`
The memory needed to be unlocked by adding `@audio - memlock unlimited`.

While fixing that issue I found that I had not properly set up realtime privileges for my user account.
Fixing that removed the original issue.

@falkTX thank you very much for the fast reply on the initial issue
